spring-cloud-dataflow-server-local/pom.xml

```diff
@@ -153,7 +153,7 @@
 							  <tag>latest</tag>
 							  <tag>${project.version}</tag>
 						  </tags>
-						<from>springcloud/openjdk:latest</from>
+						<from>java:8-alpine</from>
```

Comment:
This needs to be changed to `springcloud/openjdk:latest`

Reply:
merge mistake.